test/benchmarks/mdc/table/app.module.ts

```diff
@@ -0,0 +1,81 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {Component, NgModule, ViewEncapsulation} from '@angular/core';
+import {MatTableModule} from '@angular/material-experimental/mdc-table';
+import {BrowserModule} from '@angular/platform-browser';
+import {BasicTable} from './basic-table';
+import {
+  fiveCols, tenCols, twentyCols,
+  tenRows, oneHundredRows, oneThousandRows,
+} from './fake-table-data';
+
+/** component: mdc-table */
+
+// tslint:disable:max-line-length
```

Comment:
Should this only be around the template? Alternatively you can add some line breaks so the lines aren't so long.